Samples/Chirper/ChirperGrains/ChirperAccount.cs

```diff
@@ -31,7 +31,6 @@ public class ChirperAccountState
         public string UserAlias { get; set; }
     }
 
-    [Reentrant]
```

Comment:
Thanks Gabi, that makes sense, and I'll update it. 
Still I think this could be improved a little bit more,  as if 2 writes are waiting for a previous write to complete,  then only 1 extra write is required and not 2. Also this approach would complicate error handling coming from storage. 
Having said all that,  I think it's good enough and better than making it non reentrant when it shouldn't be